db/schema.rb

```diff
@@ -827,21 +827,21 @@
     t.index ["replaced_by_id"], name: "index_global_meter_attributes_on_replaced_by_id"
   end
 
-  create_table "good_job_processes", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+  create_table "good_job_processes", id: :uuid, default: -> { "public.gen_random_uuid()" }, force: :cascade do |t|
     t.datetime "created_at", precision: 6, null: false
```

Comment:
We've seen this in the schema after refreshing the database from live.. not sure why it's appeared, but maybe better to revert it for this PR, and handle it separately..?